src/lib/db/client-applications-store.ts

```diff
@@ -449,4 +449,16 @@ export default class ClientApplicationsStore
             }));
         }
     };
+
+    async removeInactiveApplications(): Promise<number> {
+        const rows = await this.db(TABLE)
+            .whereRaw("seen_at < now() - interval '30 days'")
```

Comment:
we could also use updated_at

Reply:
It makes sense to use both.

Reply:
yeah, good point. We can look at either seen_at which is optional and updated_at which is required

Reply:
ok I checked the code and we always have seen_at set in the app:      ```updated_at: input.updatedAt || new Date(),
        seen_at: input.lastSeen || new Date(),```